sql/moz-fx-data-shared-prod/telemetry_derived/rolling_cohorts_v1/schema.yaml

```diff
@@ -0,0 +1,67 @@
+fields:
```

Comment:
Wonder if days_since-seen can affect the cohorts. Ex. If I haven't used the browser for a while I might decide to uninstall it in the end...

Reply:
Discussed offline, might be useful in the cohort daily statistics but not ehre